akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -538,7 +542,7 @@ abstract class GraphStageLogic private[stream] (val inCount: Int, val outCount:
 
   /**
    * Automatically invokes [[cancel()]] or [[complete()]] on all the input or output ports that have been called,
-   * then stops the stage, then [[postStop()]] is called.
+   * then marks the stage as stopped.
```

Comment:
No, about not all stages cleaning up their resources in when abruptly terminated (because they did it in the handler hooks only).